pkg/apis/pravega/v1beta1/pravega.go

```diff
@@ -68,6 +68,86 @@ const (
 
 	// DefaultInfluxDBSecretMountDir is the default mountpath of influxdb secret
 	DefaultInfluxDBSecretMountDir = "/etc/influxdb-secret-volume"
+
+	// DefaultReadinessProbeInitialDelaySeconds is the default initial delay (in seconds)
+	// for the readiness probe
+	DefaultControllerReadinessProbeInitialDelaySeconds = 20
+
+	// DefaultReadinessProbePeriodSeconds is the default probe period (in seconds)
+	// for the readiness probe
+	DefaultControllerReadinessProbePeriodSeconds = 60
+
+	// DefaultReadinessProbeFailureThreshold is the default probe failure threshold
+	// for the readiness probe
+	DefaultControllerReadinessProbeFailureThreshold = 3
+
+	// DefaultReadinessProbeSuccessThreshold is the default probe success threshold
+	// for the readiness probe
+	DefaultControllerReadinessProbeSuccessThreshold = 3
```

Comment:
Do we need a success threshold of `3`?

Reply:
i didnt change the value, it was set to `3`, Should we change to `1`

Reply:
Let's see what was the reasoning for it in the initial PR

Reply:
No explanation, unfortunately
https://github.com/pravega/pravega-operator/pull/405

Reply:
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes
> successThreshold: Minimum consecutive successes for the probe to be considered successful after having failed. Defaults to 1. Must be 1 for liveness and startup Probes. Minimum value is 1.

This is a readiness probe, so there is no restriction for it to be 1.

Reply:
When the successThreshold of  was introduced initially, it was applied to querying the `scopes` API (not the health one that is being performed now) and required three successes with the period of 1 second - so it took two extra seconds and was not a big deal.
Here we are going to require three successes with a period of 60 seconds (`DefaultControllerReadinessProbePeriodSeconds = 60`, line 78), so we are asking for two extra minutes 🤔

Reply:
Here, timeoutSeconds is set to  `60`. If we set the time interval to `1` second, there is a chance that command will execute before the first one is completed

Reply:
have gone through documentation, we can keep period seconds to less than timeout seconds. Should i keep it as `10s`.  Or do you suggest to keep the default `1s`

Reply:
Let's keep everything that was not previously set explicitly at K8s defaults (10s for `periodSeconds`).

Reply:
Yes, I have set to `10s` now